back-end/bal-forum/tests/bal_forum_service_test.bal

```diff
@@ -0,0 +1,390 @@
+import ballerina/http;
+import ballerina/sql;
+import ballerina/test;
+import ballerina/time;
+import ballerina/uuid;
+import ballerinax/mysql;
+import ballerinax/nats;
+
+configurable string forumServiceUrl = "http://localhost:4000/api";
```

Comment:
Why do we have this as configurable?

Reply:
Will change that 👍 . Given that the service always runs on that fixed url, seems like no need to make it a configurable.